webxrlayers-1.bs

```diff
@@ -914,7 +915,66 @@ To <dfn>determine the layout attribute</dfn> using an {{XRTextureType}} |texture
 
 </div>
 
-ISSUE: special case UA behavior if the size causes the layout to change (ie if the requested width exceeds a limit with {{XRLayerLayout/"stereo-left-right"}})
+<div class="algorithm" data-algorithm="allocate color textures for projection layers">
+
+To <dfn>allocate color textures for projection layers</dfn> using an {{XRProjectionLayer}} |layer|, an {{XRTextureType}} |textureType|, a boolean |alpha| and a float |scaleFactor|, the user agent MUST run the following steps:
+  1. let |array| be a [=new=] array in the [=relevant realm=] of |context|.
+  1. let |context| be |layer|'s [=XRCompositionLayer/context=].
+  1. let |session| be |layer|'s [=XRCompositionLayer/session=].
+  1. let |numViews| be the number of the |session|'s [=list of views=].
+  1. Let |view| be the first entry in the |session|'s [=list of views=].
+  1. Let |width| be the width of |view|'s [=recommended WebGL texture resolution=] multiplied by |scaleFactor|.
```

Comment:
Somewhere, maybe not here but at the site that calls this algorithm, we'll want to allow the UA to clamp the `scaleFactor`. Some devices won't be able to deviate from the recommended resolution, and most will have upper and lower bounds on the acceptable texture size. I don't see that as a blocking issue for this PR, though.

Reply:
I had looked at the WebXR spec but didn't see where it limited the `scaleFactor`. Can you point me where it is?

I will open an issue.

Reply:
We should have one in the core spec but I'm not seeing it either right now. We do have a min/max clip plane that has very similar usage to what we'd want here: https://immersive-web.github.io/webxr/#minimum-near-clip-plane

We also have language in the explainer that indicates that the values may be clamped. https://github.com/immersive-web/webxr/blob/master/explainer.md#controlling-rendering-quality